habitat-baselines/habitat_baselines/config/rearrange/rl_hierarchical_oracle_nav_spot_fp.yaml

```diff
@@ -9,16 +9,32 @@
 defaults:
   - rl_hierarchical
   - /habitat/task/actions:
-    - oracle_nav_action
+    - oracle_nav_with_backing_up_action
   - override /habitat/simulator/agents@habitat.simulator.agents.main_agent: rgbd_head_rgbd_arm_agent
   - override /habitat/task/rearrange: rearrange_easy_fp
   - _self_
 
 habitat:
   task:
     actions:
-      oracle_nav_action:
+      oracle_nav_with_backing_up_action:
+        type: OracleNavWithBackingUpAction
         spawn_max_dist_to_obj: -1.0
+        # For noncylinder navmesh action
+        # If we allow the agent to do sliding
+        allow_dyn_slide: True
+        # If the agent is allowed to slide if the agent needs to do rotation
+        enable_rotation_check_for_dyn_slide: False
+        # The distance threshold of treating collision
+        collision_threshold: 0.02
+        # The navmesh offset for the action control
+        navmesh_offset: [[0.0, 0.0], [0.25, 0.0], [-0.25, 0.0]]
+        # The navmesh offset for the agent placement
+        navmesh_offset_for_agent_placement: [[0.0, 0.0], [0.25, 0.0], [-0.25, 0.0]]
+        enable_lateral_move: False
+    robot_at_thresh: 3.0
```

Comment:
Why are you modifying this?

Reply:
Spot navigation code requires considering the non-cylinder navmesh. Also changing `robot_at_thresh` allows Spot to navigate in the tight space to halt early without getting too closed to the target location.